.changeset/lemon-chicken-look.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-kubernetes': patch
+'@backstage/plugin-kubernetes-backend': patch
+'@backstage/plugin-kubernetes-common': patch
```

Comment:
seems like breaking changes for the last two here, should be `minor` then

Reply:
I've done these bumps in the new PR